crates/ra_project_model/src/lib.rs

```diff
@@ -250,7 +250,7 @@ impl ProjectWorkspace {
 
     pub fn to_crate_graph(
         &self,
-        default_cfg_options: &CfgOptions,
+        target: Option<&String>,
```

Comment:
```suggestion
        target: Option<&str>,
```

`&String`, `&Vec`, `&Box` etc are sort-of useless double indirection types. 

r=me with this change

bors d+